### PR TITLE
Avoid a first-chance exception in WindowsPlatform ImageHelper.

### PR DESCRIPTION
**Pull request by KirillOsenkov**
This should be a good safe tactical fix (WindowsPlatform only), no Xwt changes needed.

### CODE REVIEW

**Comment by KirillOsenkov** [comment]
The exception happens 75 times during startup.

File: main/src/addins/WindowsPlatform/WindowsPlatform/ImageHelper.cs [hunk 1]
```diff
@@ -43,9 +43,14 @@ public static Image GetStockIcon (this IconId stockId)
 				return null;
 			Image image;
 			if (!cachedIcons.TryGetValue (stockId, out image)) {
-				try {
-					image = Image.FromResource (typeof (ImageHelper), stockId + ".png");
-				} catch (InvalidOperationException) {
+				var resourceName = stockId + ".png";
+
+				// first check if such a resource exists to avoid a first-chance exception
+				if (typeof (ImageHelper).Assembly.GetManifestResourceStream (resourceName) != null) {
```
Comment (Therzok) [inline main/src/addins/WindowsPlatform/WindowsPlatform/ImageHelper.cs]: Do we not need to dispose the stream?

Comment (KirillOsenkov) [reply main/src/addins/WindowsPlatform/WindowsPlatform/ImageHelper.cs]: I guess so... it wouldn't hurt. Just added a commit that disposes the stream.

Comment (KirillOsenkov) [reply main/src/addins/WindowsPlatform/WindowsPlatform/ImageHelper.cs]: Dispose will run this line:
http://referencesource.microsoft.com/#mscorlib/system/io/unmanagedmemorystream.cs,277

**Comment by KirillOsenkov** [comment]
How about this PR? Can this go in?

**Pull request merged**